Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb/RBResult.py

```diff
@@ -32,6 +32,53 @@ def plot_hist(self, n_cols=3):
             self.data.state.sel(circuit_depth=circuit_depth).plot.hist(ax=ax, xticks=range(4))
         plt.tight_layout()
 
-    def plot_fidelity(self):
+    def plot_decay(self):
         fidelity = (self.data.state == 0).sum(("repeat", "average")) / (self.num_repeats * self.num_averages)
         fidelity.rename("fidelity").plot.line()
+
+    def plot_fidelity(self):
```

Comment:
Same comment here, just to clarify how plot_decay differentiates from plot_fidelity, would be nice to have a small docstring.

Reply:
Done.  Please tell me what you think